src/Shared/SpanHelper.cs

```diff
@@ -16,37 +16,37 @@ internal static class SpanHelper
     /// </summary>
     /// <param name="httpStatusCode">Http status code.</param>
     /// <returns>Resolved span <see cref="Status"/> for the Http status code.</returns>
-    public static Status ResolveSpanStatusForHttpStatusCode(int httpStatusCode)
+    public static ActivityStatusCode ResolveSpanStatusForHttpStatusCode(int httpStatusCode)
```

Comment:
Do we need this one?.

Reply:
It is used by 2 instrumentation packages. OWIN and one more.

Reply:
Got it - we should update those to use the one below.